spacy_llm/tests/tasks/test_ner.py

```diff
@@ -248,6 +248,35 @@ def test_ner_predict(cfg_str, text, gold_ents, request):
         assert pred_ent.label_ in gold_ent[1].split("|")
 
 
+@pytest.mark.external
+@pytest.mark.skipif(has_openai_key is False, reason="OpenAI API key not available")
+@pytest.mark.parametrize(
+    "text,gold_ents",
+    [
+        (
+            "Marc and Bob both live in Ireland.",
+            [("Marc", "PER"), ("Bob", "PER"), ("Ireland", "LOC")],
+        ),
+    ],
+)
+def test_llm_ner_predict(text, gold_ents):
+    """Use llm_ner factory with default OpenAI model to get NER results.
+    Note that this test may fail randomly, as the LLM's output is unguaranteed to be consistent/predictable
+    """
+    nlp = spacy.blank("en")
+    llm = nlp.add_pipe("llm_ner")
+    for ent_str, ent_label in gold_ents:
+        llm.add_label(ent_label)
+    doc = nlp(text)
+
+    assert len(doc.ents) == len(gold_ents)
+    for pred_ent, gold_ent in zip(doc.ents, gold_ents):
+        assert (
+            gold_ent[0] in pred_ent.text
+        )  # occassionally, the LLM predicts "in Ireland" instead of just "Ireland"
+        assert pred_ent.label_ in gold_ent[1].split("|")
+
+
```

Comment:
Do we need a new test for this - can't we just parametrize the other `predict` test? Overall I don't care strongly, so go ahead and merge if you prefer it this way.

Reply:
I did consider that, but the other `predict` test relies on a `config` parameter which this new one explicitely shouldn't.